utils/src/server/rpc_module.rs

```diff
@@ -130,7 +130,8 @@ impl RpcModule {
 			self.methods.insert(
 				unsubscribe_method_name,
 				Box::new(move |id, params, tx, conn| {
-					let sub_id = params.one().map_err(|e| anyhow::anyhow!("{:?}", e))?;
+					// let sub_id = params.one().map_err(|e| anyhow::anyhow!("{:?}", e))?;
```

Comment:
```suggestion
```

Reply:
My bad. Fixed this one in https://github.com/paritytech/jsonrpsee/pull/334